eng/referenceFromRuntime.targets

```diff
@@ -58,12 +60,15 @@
       </_remainingReferenceFromRuntimeWithNI>
 
       <_filteredReferencePathFromRuntimeByFileName Include="@(_referencePathFromRuntimeByFileName)"
-          Condition="'@(_referencePathFromRuntimeByFileName)' == '@(_remainingReferenceFromRuntimeWithNI)' AND '%(Identity)' != ''" />
+          Condition="'@(_referencePathFromRuntimeByFileName)' == '@(_remainingReferenceFromRuntimeWithNI)' AND '%(Identity)' != ''">
+        <Aliases>@(_remainingReferenceFromRuntimeWithNI->'%(Aliases)')</Aliases>
+      </_filteredReferencePathFromRuntimeByFileName>
 
       <_missingReferenceFromRuntime Include="@(_remainingReferenceFromRuntimeWithNI)" Exclude="@(_filteredReferencePathFromRuntimeByFileName)" />
 
       <!-- transform back to path -->
       <ReferencePath Include="@(_filteredReferencePathFromRuntimeByFileName->'%(ReferencePath)')" />
+      <ReferencePath Include="@(_filteredReferencePathFromRuntimeByFileName->'%(ReferencePath)')" Condition="'%(Aliases)' != ''" Aliases="" />
```

Comment:
@ericstj does it hurt anything on having a duplicate ReferencePath on an assembly?

Reply:
It's acceptable, but in some cases it could be a problem.  If someone specifically needed the alias to resolve a conflict in the global scope this wouldn't help them, but I am OK forcing them to alias both sides of that conflict.  I don't think we should make it too easy to write code against runtime assemblies that overlap with contract assemblies.  In those cases we should force the code to be explicit about which one it intends to use.

I think this at least warrants a comment though.